content/doc/book/blueocean/activity.adoc

```diff
@@ -18,32 +18,69 @@ repository Pipeline.
 
 image:blueocean/activity/overview.png[Overview of the Activity View, role=center]
 
-
 == Activity
 
+The default subsection of Activity Viewer shows a list of the latest
+completed or in-progress Runs of a Pipeline.
```

Comment:
I think capitalizing "Run" is fine, but the pluralization plus "of a Pipeline" makes this sentence awkward to me.

I think "in-progress Runs." would be fine, but the use of it as "of a Pipeline" makes for too many Capitalized Noun Things in the Sentence. :smiling_imp: